### PR TITLE
OS-2: Week-10: Fixed all the ambulance use cases

### PR DESCRIPTION
**Pull request by divyanshkhetan**
## Proposed changes
- Fixed the code errors in the template file.
- Increased the char limit for reason field in Ambulance_request to 500 so as to provide enough space for appropriate description of reason.
- Added migration changes to resolve previous merge conflicts and add the new change to the model.

## Types of changes

_Put an `x` in the boxes that apply_

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image
image

### CODE REVIEW

**Comment by NuthanReddy646** [comment]
Approve @akshatnema

File: FusionIIIT/templates/phcModule/appointment.html [hunk 1]
```diff
@@ -190,12 +190,12 @@
...
       {
-        var startDate = document.getElementById('start_date').value;
-        var endDate = document.getElementById('end_date').value;
+        var startDate = document.getElementById('inputdate1').value;
+        var endDate = document.getElementById('inputdate2').value;
         var start = new Date(startDate);
         var end=new Date(endDate);
         var today = new Date(new Date().setHours(0, 0, 0, 0));
-
+        console.log("Was here:::")
```
Comment (akshatnema) [inline FusionIIIT/templates/phcModule/appointment.html]:
```suggestion
```

File: FusionIIIT/templates/phcModule/appointment.html [hunk 2]
```diff
@@ -163,21 +163,21 @@
               $('#inputdate2').attr('min', maxDate);
             });
           </script>
-          <p id="dte"></p>
+          <!-- <p id="dte"></p> -->
```
Comment (akshatnema) [inline FusionIIIT/templates/phcModule/appointment.html]:
```suggestion
```

File: FusionIIIT/templates/phcModule/appointment.html [hunk 3]
```diff
@@ -144,7 +144,7 @@
               $('#inputdate1').attr('min', maxDate);
             });
           </script>
-          <p id="dte"></p>
+          <!-- <p id="dte"></p> -->
```
Comment (akshatnema) [inline FusionIIIT/templates/phcModule/appointment.html]:
```suggestion
```

File: FusionIIIT/templates/phcModule/appointment.html [hunk 4]
```diff
@@ -163,21 +163,21 @@
...
         </div>
       </div>
       <div class="field">
         <label>Reason</label>
         <div class="ui fluid input">
           <textarea class="ui textarea" rows="2" id="reason" name="reason"></textarea>
-          <p id="reas"></p>
+          <!-- <p id="reas"></p> -->
```
Comment (akshatnema) [inline FusionIIIT/templates/phcModule/appointment.html]:
```suggestion
```

**Review by akshatnema — CHANGES_REQUESTED** [review]

**Comment by akshatnema** [comment]
Review required from @PranshuNayak and @samay-rgb

File: FusionIIIT/applications/health_center/models.py [hunk 5]
```diff
@@ -158,7 +158,7 @@ class Ambulance_request(models.Model):
     date_request = models.DateTimeField()
     start_date = models.DateField()
     end_date = models.DateField(null=True, blank=True)
-    reason = models.CharField(max_length=50)
+    reason = models.CharField(max_length=500)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/health_center/models.py]: The max_length for a CharField is 255. If more words are required use TextField.

**Review by PranshuNayak — CHANGES_REQUESTED** [review]

File: FusionIIIT/applications/health_center/models.py (hunk 5 above)
Comment (divyanshkhetan) [reply FusionIIIT/applications/health_center/models.py]: @PranshuNayak Fixed the model. PTAL.

File: FusionIIIT/templates/phcModule/appointment.html (hunk 4 above)
Comment (divyanshkhetan) [reply FusionIIIT/templates/phcModule/appointment.html]: @akshatnema I've applied the suggested changes. PTAL.

**Review by PranshuNayak — DISMISSED** [review]
LGTM!

File: FusionIIIT/applications/scholarships/models.py
Comment (samay-rgb) [inline FusionIIIT/applications/scholarships/models.py:1]: Do not push indentation changes

File: FusionIIIT/applications/scholarships/migrations/0002_auto_20230411_0848.py
Comment (samay-rgb) [inline FusionIIIT/applications/scholarships/migrations/0002_auto_20230411_0848.py:1]: Remove this file from PR.

Comment (divyanshkhetan) [reply FusionIIIT/applications/scholarships/migrations/0002_auto_20230411_0848.py:1]: @samay-rgb @PranshuNayak PTAL.

**Review by PranshuNayak — APPROVED** [review]
LGTM!

**Review by samay-rgb — APPROVED** [review]

**Review by akshatnema — APPROVED** [review]

**Pull request merged**